.github/workflows/release.yml

```diff
@@ -56,18 +56,26 @@ jobs:
       - run: echo "Using VehicleApp version ${{ steps.get_version.outputs.version-without-v }} from tag"
 
       - name: Login to GitHub Container Registry
-        uses: docker/login-action@v1
+        uses: docker/login-action@v2
         with:
           registry: ghcr.io
           username: ${{ github.repository_owner }}
           password: ${{ secrets.GITHUB_TOKEN }}
 
       - id: github-repository-name-case-adjusted
         name: Prepare repository name in lower case for docker upload.
-        uses: ASzc/change-string-case-action@v1
+        uses: ASzc/change-string-case-action@v2
         with:
           string: ${{ github.repository }}
 
+      - name: Wait for CI workflow to succeed
+        uses: fountainhead/action-wait-for-check@v1.1.0
+        with:
+          checkName: Run Integration Tests (sampleapp)
```

Comment:
Here I tried (${{ matrix.component.Name }}), instead of hard coding "sampleapp", but it wasn't worked